README.md

```diff
@@ -1,5 +1,5 @@
 # <img src="docs/assets/images/logo/saros-logo.svg" alt="drawing" align="right" width="100"/> Saros - Distributed Party Programming
-[![Travis](https://travis-ci.com/saros-project/saros.svg?branch=master)](https://travis-ci.com/saros-project/saros/branches)
+[![Build](https://github.com/saros-project/saros/workflows/Build/badge.svg)](https://github.com/saros-project/saros/actions?query=workflow%3ABuild)
```

Comment:
Is this supposed to display builds for all branches?

With the previous badge, only builds of the master were displayed. Builds for other branches/PRs can routinely break without having any impact (as they might be WIP). Displaying build results for them could leave a bad impression of the state of the repo/development when visiting the README.

The [documentation](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/configuring-a-workflow#adding-a-workflow-status-badge-to-your-repository) offers options on how to do this:
```suggestion
[![Build](https://github.com/saros-project/saros/workflows/Build/badge.svg?branch=master)](https://github.com/saros-project/saros/actions?query=workflow%3ABuild+branch%3Amaster)
```